### PR TITLE
[UIE-117] Move Interactive's CSS to components package

### PR DESCRIPTION
**Pull request by nawatts**
https://broadworkbench.atlassian.net/browse/UIE-117

To support hover and focus styles, Interactive depends on some CSS. That CSS is currently located in style.css. Now that Interactive is distributed to other projects as part of the components package, that CSS should be distributed with it.

This moves the relevant CSS to packages/components/src/Interactive.js, which injects it into the DOM.

Left over from earlier revisions in this PR is a switch to a different ESLint import resolver: [eslint-import-resolver-typescript](https://www.npmjs.com/package/eslint-import-resolver-typescript). Though it's not needed for this, it looks more well maintained than the [eslint-import-resolver-custom-alias](https://www.npmjs.com/package/eslint-import-resolver-custom-alias) that we're using now so I left it in.

### CODE REVIEW

**Review by cahrens — APPROVED** [review]

**Comment by msilva-broad** [comment]
I would like us to explore a bit more if there is another way to do this that doesn't require the "shadow dependency" of the consuming app having to remember to import the css.

File: packages/components/src/Interactive.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.hover-style:hover, .hover-style:focus {
```
Comment (msilva-broad) [inline packages/components/src/Interactive.css]: to guard against css class name collisions, lets prefix these style names with something like terra-ui--hover-style

Comment (nawatts) [reply packages/components/src/Interactive.css]: Renamed in https://github.com/DataBiosphere/terra-ui/pull/4147/commits/b9b700c80794b89c76db6b093240aaeeee249ec9.

**Comment by nawatts** [comment]
https://github.com/DataBiosphere/terra-ui/pull/4147/commits/62db0add6201b27c0fd31368261d3f30b902c2f8 has Interactive inject a style element with the required CSS into the document. This removes the separate CSS file.

**Comment by msilva-broad** [comment]
> I would like us to explore a bit more if there is another way to do this that doesn't require the "shadow dependency" of the consuming app having to remember to import the css.

nice work switching to injectStyle solution.... I think the PR description needs updating to reflect the improved outcome.  No more css import requirement on consume project, YAY!

**Review by msilva-broad — APPROVED** [review]
thanks for putting in the extra effort to remove the css import requirement of the earlier implementation.  Nice work!

**Pull request merged**